2929 request appointment logging metrics

cadibemma opened the pull request:
<!-- Please read our guidelines before submitting your first PR https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md -->

## Description of change
We have limited access to production outside of logs and metrics. As part of implementation for schedule appointments we want to be able to track where things are working and at what point they stop.

## Original issue(s)
department-of-veterans-affairs/va-mobile-app#2929

<!-- Please describe testing done to verify the changes or any testing planned. -->

aherzberg reviewed (changes requested):
Looks like ticket also calls out logging for Service Eligibility and Create Appt Request endpoint. 

for FacilitiesInfo, adding the sort type to the log would be helpful to see if certain sorts are being used more than others.

jperk51 commented:
> Create Appt Request

AFAIK this endpoint hasn't be implemented yet so I figured we would just make sure it has good logging when that PR goes up. @aherzberg

cadibemma commented:
> Looks like ticket also calls out logging for Service Eligibility and Create Appt Request endpoint.
> 
> for FacilitiesInfo, adding the sort type to the log would be helpful to see if certain sorts are being used more than others.

Ok I added sort method to the log and added logging for VA eligibility
